luigi/tools/migrate.py

```diff
@@ -0,0 +1,133 @@
+"""
```

Comment:
i think according to semver we need a major revision bump right? at least a minor one seems warranted.